comp120-evaluation.tex

```diff
@@ -0,0 +1,58 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\usepackage{graphicx}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Reflective Report}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1507290}
+
+\begin{document}
+
+\maketitle
+
+\abstract{}
+
+\section{Introduction}
+Three key skills that have caused me notable difficulty during the course are reading and understanding peers' code, effectively managing my time and priorities, and asking for feedback and clarification when needed.
+
+\section{Reading and Understanding Peers' Code}
+Due to a lack of experience in reading and writing code, I have often had difficulty reading, understanding, and picking out the relevant parts when reviewing peers' code.
+
+This is an important skill to develop as it has a number of applications both in education and industry. Reading code is essential in collaborative projects, as it is necessary both to understand the code-base you are contributing to and review the contributions of your peers. It is critical that I develop this skill in preparation for the upcoming group projects.
```

Comment:
I suggest being more introspective in this section, analyzing what you found challenging rather than describing the importance of the skill.